modules/apps/object/object-rest-impl/rest-openapi.yaml

```diff
@@ -110,6 +110,11 @@ components:
                     type: string
                 externalReferenceCode:
                     type: string
+                folderExternalReferenceCode:
```

Comment:
this shouldn't be `objectEntryFolder***` ??

Reply:
You're right. I'll push a fix.